packages/backend/src/config/getDeliveryServiceProperties.ts

```diff
@@ -0,0 +1,29 @@
+import { parse } from 'yaml';
+import { existsSync, readFileSync } from 'fs';
+import * as Lib from 'dm3-lib/dist.backend';
+import { resolve } from 'path';
+
+const DEFAULT_CONFIG_FILE_PATH = resolve(__dirname, './../../src/config.yml');
```

Comment:
I think we shouldn't point to something in `src`. `src` will not be available if we install a package with yarn or npm.

Reply:
I replaced the src input with a relative one. In order to work the config.yml has to be copied after the build runs because TSC only handles .ts files. I added this to the package.json